src/patternfly/components/Switch/switch.scss

```diff
@@ -4,13 +4,16 @@
   --pf-c-switch--Height: calc(#{var(--pf-global--FontSize--md)} * #{var(--pf-global--LineHeight--md)});
   --pf-c-switch--Width: calc(#{var(--pf-c-switch--Height)} * #{var(--pf-global--golden-ratio)});
   --pf-c-switch--LineHeight: var(--pf-global--LineHeight--md);
-  --pf-c-switch__input--checked__toggle--BackgroundColor: var(--pf-global--primary-color--100);
-  --pf-c-switch__input--checked__toggle--before--Transform: translateX(calc(#{var(--pf-c-switch--Width)} - #{var(--pf-c-switch__toggle--before--Width)} - 2 * #{var(--pf-c-switch__toggle--Padding)}));
-  --pf-c-switch__input--checked__label--Color: var(--pf-global--Color--dark-100);
-  --pf-c-switch__input--not-checked__label--Color: var(--pf-global--disabled-color--100);
-  --pf-c-switch__input--disabled__label--Color: var(--pf-global--disabled-color--100);
-  --pf-c-switch__input--disabled__toggle--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-200);
-  --pf-c-switch__input--disabled__toggle--before--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+
+  // Switch Input Checked Toggle
+  --pf-c-switch__toggle--m-checked--BackgroundColor: var(--pf-global--primary-color--100);
```

Comment:
`m-checked` should just be `checked` since `checked` is the state we're targeting. This is similar to `hover` and `m-hover` (since `:hover` is a thing, that's what we use in the var name). We would only use `m-whatever` if "whatever" is _only_ a modifier class and isn't an actual state we can target in CSS. Same for `m-disabled` and `m-notChecked` - those should just be `disabled` and `not-checked`

In fact, we only need a `m-state` class for things that can't be represented in the markup, like `:hover` or `:focus`. States like `:checked` and `:disabled` can be represented in the markup with the attributes `checked` and `disabled`, so we can remove `m-checked`, `m-notChecked` and `m-disabled` from the CSS and any examples here, and just use `checked`, `not-checked` and `disabled`

Reply:
ah okay! Thanks for explaining that! made the update